podcast/2.12.m

```diff
@@ -0,0 +1,933 @@
+
+So, exact text matches and rule-based approaches–those were in use–so if you sent a message and it was a copy-paste of the same message, those would be grouped in a like-minded way. But if you even tweaked a little bit of language, that would be considered a distinct and unique note. Congress hasn't had their staff increase since, I think, 1974 or their budget for their staff increase since 1974, and is already pretty chronically underpaid.
+
+So if you're in a position in the House of Reps, you're basically the support line... You are customer support for the world's most powerful democracy, you know what I mean? But I came into recognizing that because I was one of those interns that was sorting mail. My first year of school, I took an internship on the Hill in US Senator Grassley's office, being from Iowa. He's been a Senator since 1981. He's just an institution.
+
+**Beyang Liu:**
+
+Yeah. I love that guy. He's so folksy.
+
+**Joseph Nelson:**
+
+Yeah, for listeners, if you haven't seen his tweets or even Stephen Colbert's segment on Grassley’s tweets, you absolutely have to. He writes his own tweets. He's in his upper 80s and you can kind of fill in the gaps for what you get.
+
+**Beyang Liu:**
+
+He had a whole tweet stream that I read through once that was like, here's how high the corn has grown. And it was six or seven tweets.
+
+**Joseph Nelson:**
+
+Hashtag corn watch. I'm following #cornwatch right now. There's been a bit of a drought in Grassley family farms. Nelson family farms has been a bit more lucky, but yeah, #cornwatch. This year, he introduced #soybeanwatch as well.
+
+**Beyang Liu:**
+
+Ooh, diversifying.
+
+**Joseph Nelson:**
+
+A good sequel.
+
+**Beyang Liu:**
+
+Yeah.
+
+**Joseph Nelson:**
+
+He has a war with the History Channel for playing shows that aren't history. So, whenever he turns on the TV and the History Channel isn't showing history, he tweets at them and gets upset. It's some high-quality content.
+
+**Beyang Liu:**
+
+He's got Twitter figured out to a T.
+
+**Joseph Nelson:**
+
+He really does, he really does.And then folks that take Twitter too seriously are in his replies. 
+
+Yeah, so I was actually a press intern, which meant that I was like a human Google alert for the office. Whenever the senator is mentioned, you cut out the clippings and then package it on a daily basis.
+
+**Beyang Liu:**
+
+So you were quite far from the House of Cards power struggles.
+
+**Joseph Nelson:**
+
+So, back when I was passing the Bush tax cuts, yeah.
+
+No, honestly, I actually set up a Google alert and it did my job better than I did. So that was demoralizing. But there was one day where I worked as a legislative intern, and I was sorting mail, and it just stood out to me how inefficient the process was. And not for any fault of the offices. In fact, Grass's office is famous for doing a pretty good job of being really specific and responsive to constituents.
```

Comment:
```suggestion
No, honestly, I actually set up a Google alert and it did my job better than I did. So that was demoralizing. But there was one day where I worked as a legislative intern, and I was sorting mail, and it just stood out to me how inefficient the process was. And not for any fault of the offices. In fact, Grassley's office is famous for doing a pretty good job of being really specific and responsive to constituents.
```